src/subscription/__tests__/subscribe-test.js

```diff
@@ -347,7 +346,7 @@ describe('Subscription Initialization Phase', () => {
     );
   });
 
-  it('throws an error for unknown root field', async () => {
+  it('unknown field should result in closed subscription', async () => {
```

Comment:
I think asking for an unknown field is a 400-class error code (something about the request was malformed). In that case, I think the `await createSubscription` call should throw. 

My reasonings is that if we don't throw there, we'll have a subscription that is in the closed state, which does not communicate that the request was invalid or what was wrong with it.